sidecar/.env

```diff
@@ -0,0 +1,4 @@
+SOCKETFILE=/run/spire/sockets/agent.sock
```

Comment:
see if we can change the name of this env file to `.env-test` or `.env-development`. Otherwise, it's ok as is.